content/5_Plat/Sweep_Line.mdx

```diff
@@ -38,7 +38,22 @@ number line?) never explicitly mentioned in module
 
 ### Solution 1
 
-(Divide and Conquer)
+**Complexity:** $\mathcal{O}(n\log n)$
```

Comment:
```suggestion
**Time Complexity:** $\mathcal{O}(n\log n)$
```
maybe move this below the explanation

Reply:
I may be wrong about this, but isn't time complexity usually before the explaination?